DSharpPlus/Entities/Channel/DiscordChannel.cs

```diff
@@ -414,6 +428,84 @@ public Task ModifyPositionAsync(int position, string reason = null)
             return this.Discord.ApiClient.ModifyGuildChannelPositionAsync(this.Guild.Id, pmds, reason);
         }
 
+        /// <summary>
+        /// Updates the channel parent, moving the channel to the bottom of the new category.
+        /// </summary>
+        /// <param name="parent_id">New parent ID. Will move out of parent if null.</param>
+        /// <param name="lock_permissions">Sync permissions with parent. Defaults to null.</param>
+        /// <param name="reason">Reason for audit logs.</param>
+        /// <returns></returns>
+        /// <exception cref="Exceptions.UnauthorizedException">Thrown when the client does not have the <see cref="Permissions.ManageChannels"/> permission.</exception>
+        /// <exception cref="Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
+        /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
+        /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
+        public Task ModifyParentAsync(ulong? parent_id = null, bool? lock_permissions = null, string reason = null)
```

Comment:
Should take a DiscordChannel as parameter, not ulong

Reply:
Can I do it because DiscordChannel is an Snowflake Object?

Reply:
Yes, you can get an ID from it.

Reply:
Done.